src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpFormatter.TypeNames.cs

```diff
@@ -72,7 +79,7 @@ protected override bool AppendSpecialTypeName(StringBuilder builder, Type type,
 
             if (type.IsGenericParameter)
             {
-                AppendIdentifier(builder, escapeKeywordIdentifiers, type.Name);
+                AppendIdentifier(builder, escapeKeywordIdentifiers, type.Name, out sawInvalidIdentifier);
```

Comment:
I'm guessing that's just there because they're also an early-out in the general mechanism.  I can separate them out.